src/library.js

```diff
@@ -4493,14 +4462,6 @@ LibraryManager.library = {
       flags |= 16/*EM_LOG_JS_STACK*/;
     }
 
-    var stack_args = null;
-    if (flags & 128 /*EM_LOG_FUNC_PARAMS*/) {
```

Comment:
Thanks, I think the code removals are all correct!

We should also remove the option `EM_LOG_FUNC_PARAMS` that was used here - I assume it's no longer used anywhere else? Please search for it in the entire codebase, we should update docs, remove it from the C header, etc., and also add a note to the Changelog that we are removing it. It's possible doing so will break a test, which we'll need ot fix.

I see this caused a test or two to fail, `AssertionError: 1532 != 1533` etc. I think you just need to find those tests in the python code, and update that number to the new number - what happened here is you removed a function, and we track those (so we don't remove or add any by mistake - here it's ok of course). If you get stuck there let me help out, as that test can be tricky.